SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/SensorsAnalytics.java

```diff
@@ -1135,7 +1149,6 @@ UrlEncodedFormEntity getHttpEntry(final String data) throws IOException {
       try {
         if (httpClient != null) {
           httpClient.close();
-          httpClient = null;
```

Comment:
Not needed, if users want to restart the `Consumer` after a `close()` we should offer a `start()` method for them instead.